Podcast/ExternalProfileViewController.swift

```diff
@@ -44,6 +47,8 @@ class ExternalProfileViewController: ViewController, UITableViewDataSource, UITa
         super.viewDidLoad()
         view.backgroundColor = .paleGrey
         
+        if UIScreen.main.nativeBounds.height == Constants.iPhoneXHeight { scrollYOffset -= iPhoneXOffset }
```

Comment:
let's actually make a function in `System` that is like `static func isIPhoneX()->Bool` instead of putting `UIScreen.main.nativeBounds.height == Constants.iPhoneXHeight` everywhere we can just call `System.isPhoneX()`